core/src/main/java/com/google/googlejavaformat/java/Main.java

```diff
@@ -79,7 +79,7 @@ static int main(PrintWriter out, PrintWriter err, String... args) {
       return formatter.format(args);
     } catch (UsageException e) {
       err.print(e.getMessage());
-      return 0;
+      return 1;
```

Comment:
Built-in commands use a status of 2 for usage issues. IIUC, this could use 2 to align (although any non-zero is enough)

Reply:
I wasn't aware of certain codes being used for specific scenarios; I just thought each command assigned their own meanings to non-zero codes.

After reading https://tldp.org/LDP/abs/html/exitcodes.html, `2` seems like a good fit, but it also seems like it's reserved for shell builtins?

I don't have much of an opinion on the particular number here. Happy to defer to anyone who has experience in this area

Reply:
Well, Picocli uses 2 for usage errors as well (https://picocli.info/#_exception_exit_codes), but that's a detail and 1 works just as well (in the case of GJF though, 1 can also mean an error happened formatting a file, using another code would allow distinguishing those cases)

Reply:
Cool, I've updated with exit code 2 and included a comment